modules/ROOT/pages/_partials/sync/js-client-getting-started.adoc

```diff
@@ -27,9 +27,9 @@ import {
 
 == Configuration
 
-To provide custom configuration to the client, the following options are available. If you wish, these are also available by using the `DataSyncConfig` interface from the SDK.
+An instance of `DataSyncConfig` can be used to configure the Voyager client. At a minimum, the `httpUrl` is required and it is the url of the Voyager server application. If the client app uses subscription as well, then the `wsUrl` field is required too. The example beblow demostrates how to create a new instance of `DataSyncConfig`:
```

Comment:
This language seems a little strange to me. `DataSyncConfig` is not a class. Calling this an instance of DataSyncConfig seems really strange to me. Also, the bigger question here is should we centring our docs around typescript usage or JavaScript usage?

To me it's very strange to go with this approach for a couple of reasons.

* Apollo Client (and server) are not specifically designed for TypeScript users. It's usable in JavaScript and all of their docs are JavaScript focused.
* Despite any personal opinions on TypeScript vs JavaScript, JavaScript is still massively dominant and will be for a long time.

@wei-lee @wtrocki what do you guys think? Perhaps we should kick off a thread on the ML?

Reply:
@darahayes This might be myself in README.md of the docs. Happy to go with JS.

Reply:
yes, I noticed it as well, all our server side example code is written in js, but client side samples are in typescript. I think we should keep them consistent. I will update the sample code to use js instead.

Reply:
@darahayes Ideally we could have TypeScript and JavaScript. However due to time and possibly docs limitations we may just stick with JavaScript for the moment. README.md points to actual implementation where here we can actually point to API docs. 

BTW:
`Calling this an instance of DataSyncConfig seems really strange to me`

It is common to name realization of interface an instance, but this is TypeScript vocabulary that may be abstract to some.